source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -89,47 +102,89 @@ Extractor::Extractor(const envoy::api::v2::filter::http::Extraction &extractor)
   }
 }
 
+absl::optional<std::string>
+Extractor::getDynamicMetadataValue(Http::StreamFilterCallbacks &callbacks) const {
+  Upstream::ClusterInfoConstSharedPtr ci = callbacks.clusterInfo();
```

Comment:
`callbacks.streamInfo().dynamicMetdata()` might be better
Jacob mentioned that clusterInfo may be statically defined

Reply:
Consider using setMetdata filter in e2e/integration testing

Reply:
No longer relevant, as the functionality adding support for dynamic metadata extractor inputs has been removed from this PR